samples/java-valueentity-customer-registry/src/main/java/customer/Main.java

```diff
@@ -27,6 +27,18 @@ public static AkkaServerless createAkkaServerless() {
                     "customerByName",
                     CustomerDomain.getDescriptor())
             // end::register[]
+            .registerView(
```

Comment:
`registerView` must be some leftover, it should be `.register(ViewProvider)`. Can you remove the old `registerView` and fix any references.

Reply:
In the eventing sample I found it easiest to update main by renaming it to something else (but retaining the Main in pom.xml) and regenerating it with codegen and then copying over the annotations.

Reply:
Thanks, will do.